inc/cavalcade_runner_to_cloudwatch/namespace.php

```diff
@@ -59,7 +59,7 @@ function on_job_completed( Worker $worker, Job $job ) {
 }
 
 /**
- * Called when a job completed or failed via Cavalcade, and sends a Dureaction metric to CloudWatch.
+ * Called when a job completed or failed via Cavalcade, and sends a Duraction metric to CloudWatch.
```

Comment:
Wow, good try?
```suggestion
 * Called when a job completed or failed via Cavalcade, and sends a Duration metric to CloudWatch.
```

Reply:
Why do it correctly when someone will correct it for you?!